$OpenBSD$

Fix build when fileno(3) is a macro.
See https://github.com/taisei-project/taisei/issues/165

Can be dropped for taisei>1.3

Index: src/util/compat.h
--- src/util/compat.h.orig
+++ src/util/compat.h
@@ -11,6 +11,10 @@
 
 #include "taisei.h"
 
+#ifdef TAISEI_BUILDCONF_HAVE_POSIX
+#define _POSIX_C_SOURCE 200809L
+#endif
+
 // Common standard library headers
 #include <complex.h>
 #include <ctype.h>
